pages/services.js

```diff
@@ -94,6 +95,10 @@ const Services = () => (
         ]}
       />
     </TableOfContents>
+
+    <AccordionItem title="Footnotes">
+      <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
+    </AccordionItem>
```

Comment:
Carbon doesn't allow `<AccordionItem>` without a corresponding `<Accordion>`. Not sure if DDS changes such guideline?

Reply:
Thanks for this @asudoh, I didn't know about the guidelines, but the code of [FooterNav](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/react/src/components/Footer/FooterNav.js#L8) and [FooterNavGroup](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/react/src/components/Footer/FooterNavGroup.js#L8) are using as you said so I will fix it.